src/views/Settings.vue

```diff
@@ -410,6 +461,53 @@ export default defineComponent({
       }
       // Fetch the updated configuration
       await this.getRerouteFulfillmentConfiguration(config.settingTypeEnumId);
+    },
+    async updateNotificationPref(enumId: string, event: any) {
+      try {
+        const ofbizInstanceName = this.userProfile.ofbizInstanceName
```

Comment:
This should be "omsInstanceName" we can't make this specific to Ofbiz

Reply:
Yes, got it. Will make changes to the Get Use Profile API and the respective occurrences on the UI side. Currently, this variable does not hold an OMS instance name, it is going to be a unique keyword. Currently "hotwax" is getting set for all the instances during deployment and used internally for some operations.

I had a discussion with Deepak Sir to set this to some unique value (most probably the OMS instance) so that the firebase topic can be created uniquely per OMS instance.